test/IRGen/objc_methods.swift

```diff
@@ -25,16 +25,24 @@ class Foo: Fooable {
   @IBAction func garply(_: AnyObject?) {}
   @objc func block(_: (Int) -> Int) {}
   @objc func block2(_: (Int,Int) -> Int) {}
-
+  
   @objc func takesString(_ x: String) -> String { return x }
   @objc func takesArray(_ x: [AnyObject]) -> [AnyObject] { return x }
   @objc func takesDict(_ x: [NSObject: AnyObject]) -> [NSObject: AnyObject] { return x }
   @objc func takesSet(_ x: Set<NSObject>) -> Set<NSObject> { return x }
-
+  
```

Comment:
Nitpick: can you avoid changing whitespace elsewhere in the file? (We also generally try to clear out trailing whitespace even in indented blocks, but even without that it's better if the diff is as small as possible.)

Reply:
Yes, I can change that. I originally had my editor set to clear trailing whitespace, but then found that lib/IRGen/GenObjC.cpp has trailing whitespace and reversed course.